vendor/github.com/hashicorp/go-azure-sdk/resource-manager/network/2022-07-01/applicationgateways/model_applicationgatewaywebapplicationfirewallconfiguration.go

```diff
@@ -12,6 +12,7 @@ type ApplicationGatewayWebApplicationFirewallConfiguration struct {
 	MaxRequestBodySize     *int64                                         `json:"maxRequestBodySize,omitempty"`
 	MaxRequestBodySizeInKb *int64                                         `json:"maxRequestBodySizeInKb,omitempty"`
 	RequestBodyCheck       *bool                                          `json:"requestBodyCheck,omitempty"`
+	RequestBodyEnforcement *bool                                          `json:"requestBodyEnforcement,omitempty"`
```

Comment:
We can't modify files under the vendor folder since this will result in compilation failures 
```suggestion
```